bin/condor_submit

```diff
@@ -50,7 +50,7 @@ COLLECTOR_HOST = htcondor.param.get("COLLECTOR_HOST", "gpcollector03.fnal.gov")
 class StoreGroupinEnvironment(argparse.Action):
     """Action to store the given group in the GROUP environment variable"""
 
-    def __call__(self, parser, namespace, values, option_string=None):
+    def __call__(self, parser:argparse.ArgumentParser, namespace:Any, values:Dict[str,str], option_string=None:Union[None,str]):
```

Comment:
Any reason the `namespace` is type `Any` when we're using this with `argparse`, so the expected type is `argparse.Namespace`?

Reply:
Mostly I didn't chase down what type it actually was... I can update that.